docker-compose-staging.yml

```diff
@@ -7,7 +7,7 @@ services:
     container_name: telescope_staging
     build:
       context: .
-      dockerfile: Dockerfile
+      dockerfile: staging.Dockerfile
```

Comment:
Can we leave this as `Dockerfile` for the default/dev case?  In staging/prod we can do more work to pick the right name.